.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: "🐛 Bug report"
+about: Report a bug found while using Valora.
+title: ''
+labels: ''
+assignees: ''
+---
+
+<!-- 👋 Use the template below to report a bug. Fill in as much info as possible. 
+
+Have an issue that needs support e.g. transaction issues specific to an account? Contact our support team in-app or 👉 https://valoraapp.com/support
+
+Have a question about development? Join our Discord 👉 https://discord.gg/7tKnCbHv5j
+
+As an open source project - it may take some time for your issue to be addressed. Please be patient and we will respond as soon as we can. 🙏 -->
+
+### Current behavior
+<!-- A description including screenshots, videos, etc. -->
+
+### Desired behavior
+<!-- Remember, we are not familiar with the application you're testing, so please provide a clear description of what should happen.-->
```

Comment:
I'm a little confused by the phrasing `we are not familiar with the application you're testing`. Isn't this for Valora?